src/state/store.ts

```diff
@@ -22,7 +22,7 @@ function bindMiddleware(middleware: Middleware[]) {
 const persistConfig = {
   key: 'root',
   storage: AsyncStorage,
-  blacklist: ['wallets', 'authenticators', 'authentication', 'electrumX', 'filters', 'toastMessages'],
+  blacklist: ['wallets', 'authenticators', 'authentication', 'electrumX', 'filters', 'toastMessages', 'notification'],
```

Comment:
I would rather keep notifications in redux, why not?

Reply:
not really, feel that will change all in my logic :)

Reply:
in fact it is stored in redux because @PanRadaCloudbest made typo :P it should be `notifications`, ` s` is missing

Reply:
:) true my fault

Reply:
done